### PR TITLE
f-cookie-banner@v0.3.0 - remove f-globalisation mixin

### PR DESCRIPTION
**Pull request by simonsandersje**
---

- Update tests to use js-cookie.
- Update legacy cookie value
- Remove f-globalisation mixin

---

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [x] Internet Explorer 11

### List any other browsers that this PR has been tested in:

- [x] Edge
- [x] FireFox

### CODE REVIEW

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue [hunk 1]
```diff
@@ -205,7 +202,7 @@ export default {
          */
         checkCookieBannerCookie () {
             if (this.legacyBanner) {
-                this.shouldHideBanner = CookieHelper.get(this.legacyConsentCookieName) === 2;
+                this.shouldHideBanner = CookieHelper.get(this.legacyConsentCookieName) === '130315';
```
Comment (DamianMullins) [inline packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:205]: What does "130315" refer to?

Comment (simonsandersje) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:205]: @DamianMullins I think it refers to a cookie policy version number:
image

Comment (simonsandersje) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:205]: But this is for the legacy cookie banner functionality, which we are replacing, so I hard-coded it cos we'll be deleting it soon anyway

Comment (DamianMullins) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:205]: Thanks @simonsandersje

File: packages/components/organisms/f-cookie-banner/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-cookie-banner",
   "description": "Fozzie Cookie Banner – Cookie Banner",
-  "version": "0.2.0",
+  "version": "0.2.1",
```
Comment (DamianMullins) [inline packages/components/organisms/f-cookie-banner/package.json]: As there's a fair bit of work in this PR maybe it should be a minor version bump?

Comment (simonsandersje) [reply packages/components/organisms/f-cookie-banner/package.json]: @DamianMullins Agreed

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue [hunk 3]
```diff
@@ -26,44 +26,41 @@
...
-                        <template #cookiePolicy>
-                            <a
-                                :href="$t('cookiePolicyLinkUrl')"
-                                target="_blank">
-                                <span>{{ $t('cookiePolicyLinkText') }}</span>
-                            </a>
-                        </template>
-                    </i18n>
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ copy.textLine3 }}
```
Comment (minrwhite) [inline packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:41]: I'd be tempted to use a [custom filter](https://v1.vuejs.org/guide/custom-filter.html) for this component only to scaffold the link into `textLine3` in its existing form and avoid having to split it into `textLine3` and `textLine4` in all the language files, only to merge them back together again when `f-globalisation` has been fixed. Happy to accept like this though.

**Review by minrwhite — APPROVED** [review]

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue (hunk 3 above)
Comment (simonsandersje) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:41]: @benwhite-deltas It was done like that, but in pulling out the f-globalisation mixin, I opted to revert to the way I'd done it before

**Review by AlixRichards — APPROVED** [review]

**Pull request merged**